src/hooks/form-date-picker/useFormDatePickerProps.tsx

```diff
@@ -64,6 +64,7 @@ export default function useFormDatePickerProps({
     [icon],
   )
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why was this needed for, that should be used later on

Reply:
ah whoops, I'll remove on master and you can 👍